src/main/resources/static/css/style2.css

```
@@ -274,3 +280,183 @@ main{
+    transition: all 0.2s ease;
+}
+
+.user-dropdown.show {
+    display: block;
+    opacity: 1;
+    transform: translateY(0);
+}
+
+.user-info {
+    margin-bottom: 10px;
+    margin: 0 0 14px 0;
```

Stylelint (17.9.0)

[error] Overridden property "margin-bottom" by shorthand "margin" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)